symphony-bdk-core/src/test/java/com/symphony/bdk/core/config/BdkConfigLoaderTest.java

```diff
@@ -0,0 +1,53 @@
+package com.symphony.bdk.core.config;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.symphony.bdk.core.config.model.BdkConfig;
+import com.symphony.bdk.core.exceptions.BdkConfigException;
+import org.junit.jupiter.api.Test;
+
+import java.io.InputStream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class BdkConfigLoaderTest {
+
+    @Test
+    public void loadFromYamlInputStreamTest() throws JsonProcessingException, BdkConfigException {
+        InputStream inputStream = BdkConfigLoaderTest.class.getResourceAsStream("/config.yaml");
+        BdkConfig config = BdkConfigLoader.loadFromInputStream(inputStream);
+        assertEquals(config.getBot().getUsername(), "tibot");
+    }
+
+    @Test
+    public void loadFromJsonInputStreamTest() throws JsonProcessingException, BdkConfigException {
+        InputStream inputStream = BdkConfigLoaderTest.class.getResourceAsStream("/config.json");
+        BdkConfig config = BdkConfigLoader.loadFromInputStream(inputStream);
+        assertEquals(config.getBot().getUsername(), "tibot");
+    }
+
+    @Test
+    public void loadFromYamlFileTest() throws JsonProcessingException, BdkConfigException {
+        String configPath = System.getProperty("user.dir") + "/src/test/resources/config.yaml";
+        BdkConfig config = BdkConfigLoader.loadFromFile(configPath);
+        assertEquals(config.getBot().getUsername(), "tibot");
+    }
+
+    @Test
+    public void loadFromJsonFileTest() throws JsonProcessingException, BdkConfigException {
+        String configPath = System.getProperty("user.dir") + "/src/test/resources/config.json";
```

Comment:
Does it work in IDEA? not sure the default current folder is the module's folder